tests/test_resources.py

```diff
@@ -146,3 +146,25 @@ def test_worker_detect_gpus_from_env(hq_env: HqEnv):
         ["worker", "hwdetect"], env={"CUDA_VISIBLE_DEVICES": "1,3"}
     )
     assert "gpus: List(1,3)" in resources
+
+
+def test_array_submit_resources(hq_env: HqEnv):
```

Comment:
Please change this test to instead check the output of `hq task info`, not the output of the submit command.